src/ServerConfigParams.ts

```diff
@@ -90,9 +100,32 @@ export function verifyServerConfiguration({
   if (!app.url) throw new Error('missing param: url');
   if (!app.port) throw new Error('missing param: port');
   if (!app.workdir) throw new Error('missing param: workdir');
+
+  const {
+    disableSponsoredTx,
+    gasFeePercentage,
+    transferFeePercentage,
+    fixedUsdFee,
+  } = app;
+
+  //disableSponsoredTx should be defined
+  if (![true, false].includes(disableSponsoredTx))
+    throw new Error(ERROR_DISABLE_SPONSOR_TX_NOT_CONFIGURED);
+
+  if (disableSponsoredTx) {
+    //If any of the fee parameters is a number it can not be a negative value
+    if (typeof gasFeePercentage == 'number' && gasFeePercentage < 0)
+      throw new Error(ERROR_GAS_FEE_PERCENTAGE_NEGATIVE);
+
+    if (typeof transferFeePercentage == 'number' && transferFeePercentage < 0)
```

Comment:
I think that setting this with a value lower than 0 is the only way to disable it, unless we decide to set it as `undefined` in default.json5. (that would probably be much better)

Reply:
Agreed on put this on a separated task.